src/scripts/main.js

```diff
@@ -1,8 +1,19 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const person of people) {
```

Comment:
Ось тут можна так в майбутніх тасках
```suggestion
for (const { name, sex, born, died } of people) {
```
відповідно далі .персон можна стерти

Reply:
спасибо Юр, да, действительно, ты прав)